test/StreamJsonRpc.Tests/MarshalableProxyTests.cs

```diff
@@ -93,6 +93,11 @@ public interface IMarshalable : INonMarshalable
     {
     }
 
+    [RpcMarshalable(CallScopedLifetime = true)]
+    public interface IMarshalableWithCallScopedLifetime : IMarshalable
```

Comment:
Am I right that, if I had
```cs
    [RpcMarshalable]
    public interface IMarshalableWithNonCallScopedLifetime : IMarshalableWithCallScopedLifetime
```
it would not be call-scoped?

Reply:
correct.